plugin/src/main/java/com/denizenscript/denizen/utilities/Utilities.java

```diff
@@ -587,18 +587,14 @@ public static <T> T elementToEnumlike(ElementTag element, Class<T> type) {
     }
 
     public static <T> T elementToEnumlike(ElementTag element, Class<T> type, boolean showWarning) {
-        T value = (T) element.asEnum((Class) type);

```

Comment:
please never don't call `asEnum` here. It is a stable high-performance preferable way to do the call. The other logic is slower and dirtier.